po/am.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: gnome-software\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-08-13 11:02+0000\n"
+"POT-Creation-Date: 2019-08-13 11:55+0000\n"
```

Comment:
Perhaps the changes in “Reapply old downstream translations” which just change the `POT-Creation-Date` could be dropped? But there’s not much harm in keeping them either, and it does make this commit less automatable. Up to you.

Reply:
This is added by the "update .po files to normalize formatting" step. It would indeed be nice to avoid but I'm not quite sure the best way to glue these tools